### PR TITLE
rent

### PR DESCRIPTION
**Pull request by serhii-vorona**

### CODE REVIEW

**Review by cxrmx — APPROVED** [review]
good job)

**Review by Anna-Tkachenko — CHANGES_REQUESTED** [review]
You didn't create any variables, check point 1 from checklist
I also sent message in slack about creating variables in this task yesterday

File: src/calculateRentalCost.js [hunk 1]
```diff
@@ -29,6 +29,28 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const oneDayPay = 40;
+  let rentCost = 0;
+  const more7DaysSale = 50;
+  const more3DaysSale = 20;
+
```
Comment (Anna-Tkachenko) [inline src/calculateRentalCost.js:36]: Check point 5 in checklist about variables naming

File: src/calculateRentalCost.js [hunk 2]
```diff
@@ -29,6 +29,28 @@
...
+  if (days < 3) {
+    rentCost = days * oneDayPay;
+
+    return rentCost;
+  }
```
Comment (Anna-Tkachenko) [inline src/calculateRentalCost.js:41]:
```suggestion
  if (days < 3) {
    return days * oneDayPay;
  }
```

File: src/calculateRentalCost.js [hunk 3]
```diff
@@ -29,6 +29,28 @@
...
+
+  if (days >= 3 && days < 7) {
+    rentCost = days * oneDayPay - more3DaysSale;
```
Comment (Anna-Tkachenko) [inline src/calculateRentalCost.js:44]: Don't repeat yourself: create separate variable for days * onePerDay value

**Review by Anna-Tkachenko — CHANGES_REQUESTED** [review]
Check comments

**Review by Anna-Tkachenko — APPROVED** [review]

**Pull request closed without merging**